[python-lxc] set_config_item() inserts empty value

```python
>>> import lxc
>>> c = lxc.Container('test')
>>> c.set_config_item('lxc.network.0.ipv4', ['192.168.1.1/24', '10.0.0.1/8'])
True
>>> c = lxc.Container('test')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/Documents/git/.venv/lib/python3.5/site-packages/lxc/__init__.py", line 157, in __init__
    _lxc.Container.__init__(self, name)
RuntimeError: Container_init:lxc.c:531: error during init for container 'test'.
```

```bash
elie@ubuntu$ cat /var/lib/lxc/test/config
[...]
lxc.network.0.ipv4 =
lxc.network.0.ipv4 = 192.168.1.1/24
lxc.network.0.ipv4 = 10.0.0.1/8
```

Is there a workaround?

--- a/src/lxc/confile.c
+++ b/src/lxc/confile.c
@@ -870,6 +870,9 @@ static int config_network_ipv4(const char *key, const char *value,
 	struct lxc_list *list;
 	char *cursor, *slash, *addr = NULL, *bcast = NULL, *prefix = NULL;
 
+	if (!value || !strlen(value))
+		return lxc_clear_config_item(lxc_conf, key);
+
 	netdev = network_netdev(key, value, &lxc_conf->network);
 	if (!netdev)
 		return -1;
@@ -996,6 +999,9 @@ static int config_network_ipv6(const char *key, const char *value,
 	struct lxc_list *list;
 	char *slash,*valdup;
 	char *netmask;
+
+	if (!value || !strlen(value))
+		return lxc_clear_config_item(lxc_conf, key);
 
 	netdev = network_netdev(key, value, &lxc_conf->network);
 	if (!netdev)

